Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml

```diff
@@ -85,8 +85,11 @@
       - SimpleHostile
   - type: HTN
     rootTask:
-      task: DragonCarpCompound
+      task: KillerTomatoCompound
   - type: NPCImpritingBehaviour
+    whitelist:
+      components:
+      - Actor
```

Comment:
@metalgearsloth I need to know a better criterion that selects only player-driven entities

Reply:
Not only humanoids

Reply:
humanoidappearance probably.